iib/web/api_v1.py

```diff
@@ -90,6 +92,27 @@ def get_builds():
     return flask.jsonify(response)
 
 
+@api_v1.route('/healthcheck')
+def get_healthcheck():
+    """
+    Respond to a health check.
+
+    :rtype: str
+    """
+    # Test DB connection
+    try:
+        db.engine.execute(text("SELECT 1"))
+    except Exception:
+        return flask.jsonify({"text": "Database health check failed"})
```

Comment:
This will return a 200 status code. I would not expect this if the healthcheck failed.

My suggestion is to raise an `IIBError` exception and this will automatically return a `500` status code and the JSON of `{"error": exception_msg}`.

I'm not sure 500 is the best error code but it's not a bad one in this case. I'm open to suggestions.

Reply:
Done.